client/src/components/Questions/QuestionsTab/QuestionsTab.css

```diff
@@ -0,0 +1,12 @@
+.tab {
```

Comment:
Your css styling looked perfect in our meeting. Gotta teach me how to do it lol.